hibernate-core/src/main/java/org/hibernate/engine/jdbc/connections/internal/DriverManagerConnectionProviderImpl.java

```diff
@@ -384,7 +384,8 @@ private void startIfNeeded() {
 				if ( active ) {
 					return;
 				}
-				executorService = Executors.newSingleThreadScheduledExecutor();
+				ValidationThreadFactory vtf = new ValidationThreadFactory();
+				executorService = Executors.newSingleThreadScheduledExecutor(vtf);
```

Comment:
A reminder for me: we need to reformat the new code to comply with the Hibernate formatting rules.

Reply:
@vladmihalcea can you also reformat the added code properly too if you're working on this PR? Thanks!

Reply:
@gsmet natürlich.